forest/oak/base.py

```diff
@@ -592,7 +624,7 @@ def run(study_folder: str, output_folder: str, tz_str: str = None,
                     bout_time = [t_i.to_pydatetime() for t_i in bout_time[:-1]]
 
                     # find observations in this bout
-                    acc_ind = np.isin(t_shifted, bout_time)
+                    acc_ind = ismember(t_shifted, bout_time)
```

Comment:
How about something like:
```
acc_ind = [observation in bout_time for observation in t_shifted]
```
Also, it'd be great to give `acc_ind` a less cryptic name.